mod/messages/languages/en.php

```diff
@@ -15,7 +15,6 @@
 	'messages:title' => "Subject",
 	'messages:to:help' => "Write recipient's username here.",
 	'messages:replying' => "Message replying to",
-	'messages:inbox' => "Inbox",
```

Comment:
If you look 6 lines above, you will see that it's there already. https://github.com/Elgg/Elgg/pull/9847/files#diff-a884bee3e6e6d9e615975c1a12bca57cL12